recipes/xkbcommon/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+from conans import ConanFile, tools
+from conans.errors import ConanException
+
+
+class XkbcommonConan(ConanFile):
+    name = "xkbcommon"
+    description = "keymap handling library for toolkits and window systems"
+    topics = ("conan", "xkbcommon", "keyboard")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/xkbcommon/libxkbcommon"
+    license = "MIT"
+    settings = {"os": "Linux"}
```

Comment:
I prefer moving it to configure. It's not the first time we have a similar case. The advantage there is throwing a ConanInvalidConfiguration and write a specific and detailed message.